sql/hive/src/test/resources/ql/src/test/queries/clientpositive/smb_mapjoin_25.q

```diff
@@ -19,7 +19,7 @@ select * from (select a.key from smb_bucket_1 a join smb_bucket_2 b on (a.key =
 
 set hive.optimize.bucketmapjoin=true;
 set hive.optimize.bucketmapjoin.sortedmerge=true;
-set hive.mapred.reduce.tasks.speculative.execution=false;
+set hive.mapreduce.job.reduces.speculative.execution=false;
```

Comment:
looks like`hive.mapred.reduce.tasks.speculative.execution` in the [Hive wiki|https://cwiki.apache.org/confluence/display/Hive/Configuration+Properties]. 

But probably best to pull in a Hive developer, maybe @jcamachor. Jesus: could you look at thee hive config options and make sure they are the current set?